modules/guardduty-member-account/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "guardduty_member_detector" {
```

Comment:
We can remove the redundancy in the output names...

```suggestion
output "detector" {
```

Reply:
Removed GuardDuty from all of the output names.